WebInstall/explainer_same_domain.md

```diff
@@ -62,21 +62,23 @@ The **`navigator.install()` method can overlap with some functionality of `befor
 
 On UAs that support prompting, the threshold for `navigator.install()` to resolve on same-origin installations uses the same checks that `onbeforeinstallprompt` currently has for prompting (if required by the UA). The promise doesn't resolve unless the *installability criteria* is met. *Note that the criteria defined by UAs varies and can be that there is NO criteria*.
 
-When called on the same domain, the **`install()` method will trigger/open the prompt for installation the same way that using `onbeforeinstallprompt` does right now for browser that prompts.** If the domain is not installable content, then the promise returns a `DOMException` of type 'AbortError'.
+When called on the same domain, the **`install()` method will trigger/open the prompt for installation the same way that using `onbeforeinstallprompt` does right now for browser that prompts.** If there is an error with the installation, then the promise returns a `DOMException` of type 'AbortError'. 
+
+*Any same-origin content can be installed even if it is **NOT** an application.*
 
 
 ## Proposed Solution
 
 ### The `navigator.install` method
 
-To install a web site/app, the site/app would use the promise-based method `navigator.install([<params>]);`. This method will:
+To install a web site/app, the site/app would use the promise-based method `navigator.install(manifest_id, [<params>]);`. This method will:
```

Comment:
You reference `https://w3c.github.io/manifest/#id-member` in the cross-origin explainer as an explanation for what `manifest_id` actually means, but you don't have that reference anywhere in this doc.